src/pages/index.js

```diff
@@ -67,17 +67,16 @@ carouselList.forEach((carouselElement) => {
 const formValidator = new FormValidator(settingsForValidation, formElement);
 formValidator.enableValidation();
 
+const api = new Api({ baseUrl, headers });
 donateBtn.addEventListener('click', (e) => {
   e.preventDefault();
   e.target.textContent = 'Thank You!';
 
-  const api = new Api({ baseUrl, headers });
-
   const data = {
```

Comment:
I think this syntax might be better:
```suggestion
    const data = {
    Name: nameInput.value,
    Surname: surnameInput.value,
    Email: emailInput.value,
    Pledge: selectCheckRadioValue(Array.from(donationAmount))
  };
```